tsconfig.json

```diff
@@ -13,6 +13,5 @@
     "jsx": "react",
     "jsxFactory": "h"
   },
-  "include": ["src", "types/jsx.d.ts"],
-  "exclude": ["node_modules"]
+  "include": ["src", "node_modules/@esri/calcite-components/dist/types/components.d.ts"]
```

Comment:
It possible to exclude all of node modules with this as an exception? Worried what the side-effects of not having node_modules excluded would be.